src/System.Security.Cryptography.Cng/pkg/System.Security.Cryptography.Cng.pkgproj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Cng.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.1</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Security.Cryptography.Cng.csproj" />
+    
+    <!-- All elements from previous packages that will be included in the newly built package -->
+    <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
+    <HarvestIncludePaths Include="ref/netstandard1.3" />
```

Comment:
>But then there's the issue that the targets would need an older version of the ref cs file from which to produce the PSNE assembly... unless PSNE generation looks at the already produced dll.

Got that working, but then the issue is that the packaging targets validation doesn't like the package including assemblies for any netstandard whenever netcoreapp is included inbox.

```
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework .NETCoreApp,Version=v2.0 should support System.Security.Cryptograp
hy.Cng inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.6/System.Security.Cryptography.Cng.dll.  You ma
y need to add <InboxOnTargetFramework Include="netcoreapp2.0" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng
\pkg\System.Security.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework .NETCoreApp,Version=v2.0 should support System.Security.Cryptograp
hy.Cng inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.6/System.Security.Cryptography.Cng.dll.  You ma
y need to add <InboxOnTargetFramework Include="netcoreapp2.0" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng
\pkg\System.Security.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework .NETCoreApp,Version=v2.0 should support System.Security.Cryptograp
hy.Cng inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.6/System.Security.Cryptography.Cng.dll.  You ma
y need to add <InboxOnTargetFramework Include="netcoreapp2.0" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng
\pkg\System.Security.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
  D:\git\corefx\Tools\Packaging.targets(1097,5): error : Framework UAP,Version=v10.1 should support System.Security.Cryptography.Cng
inbox but contained a implementation assemblies: runtimes/win/lib/netstandard1.4/System.Security.Cryptography.Cng.dll.  You may need
to add <InboxOnTargetFramework Include="uap10.1" /> to your project. [D:\git\corefx\src\System.Security.Cryptography.Cng\pkg\System.S
ecurity.Cryptography.Cng.pkgproj]
```